### PR TITLE
Tidying Up Custom Server Selection

### PR DESCRIPTION
**Pull request by ciaranmul**
Really looking forward to having a native iOS client for Jellyfin. I felt like I could offer some small improvements.

- Cleaning up formatting
- Providing safer encapsulation by making things private and moving things to smaller scope
- Adding extension method to check if a String is not blank as well as not empty
- Adding functions for resetting state on the ViewController
- Stopping autocapitalisation on the server address TextField
- Removing force unwrapping for safety

### CODE REVIEW

**Comment by anthonylavado** [comment]
Thank you for this! I'll try to review it soon.

**Comment by s3rverro0m** [comment]
I was thinking about moving the UI to use SwiftUI which will only work for iOS 13+ but it’ll be cleaner and simpler. I haven’t gotten around to writing it all yet though.

File: Jellyfin Player/Extensions/StringExtension.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//
```
Comment (thornbill) [inline Jellyfin Player/Extensions/StringExtension.swift:1]: All files need to start with the MPL license header

Comment (anthonylavado) [reply Jellyfin Player/Extensions/StringExtension.swift:1]:
```suggestion
//
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
```

**Review by anthonylavado — CHANGES_REQUESTED** [review]
Just apply the change as per @thornbill and mine's suggestion, and we should be good to go!

**Review by anthonylavado — APPROVED** [review]

**Pull request merged**